resteasy-core/src/main/java/org/jboss/resteasy/plugins/providers/sse/SseEventOutputImpl.java

```diff
@@ -306,43 +306,38 @@ private CompletionStage<Void> internalWriteEvent(final OutboundSseEvent event) {
 
     private BuiltResponse createResponse() {
         BuiltResponse jaxrsResponse;
-        if (state.get() == CLOSED) {
-            jaxrsResponse = (BuiltResponse) Response.noContent().build();
```

Comment:
Why did we remove this? Return "OK" when the response is closed doesn't seem right and changes behavior quite a bit.